pages.de/common/ugrep.md

```diff
@@ -0,0 +1,36 @@
+# ugrep
+
+> Ultraschnelles Suchtool mit Abfrage-TUI.
+> Weitere Informationen: <https://github.com/Genivia/ugrep>.
+
+- Ausführen das interaktives Suchtool TUI (CTRL-Z um Hilfe):
+
+`ugrep --query`
+
+- Suche rekursiv alle Dateien die ein reguläres Ausdrucksmuster enthalten:
+
+`ugrep "{{SuchMuster}}"`
```

Comment:
```suggestion
`ugrep "{{suchmuster}}"`
```

Reply:
But it's `Suchmuster`, not `Such Muster`. It's a single word

Reply:
Sure, same as Dutch grammar :-)